src/util/renderers/abstract.renderer.ts

```diff
@@ -732,5 +733,23 @@ export abstract class Renderer {
     return parsoidDoc
   }
 
+  protected injectHeader(content: string, articleDetail: any): string {
```

Comment:
Please rename to `injectH1TitleToHtml()`, adding a bit of documentation why with need it would be usefull as well.

Reply:
Done.